gowin/arch.cc

```diff
@@ -551,17 +554,20 @@ void Arch::read_cst(std::istream &in)
             if (belname != nullptr) {
                 std::string bel = IdString(belname->src_id).str(this);
                 it->second->setAttr(IdString(ID_BEL), bel);
-            } else if (std::regex_match(pinline, match_pinloc, iobelre)) {
-                // may be it's IOx#[AB] style?
-                Loc loc = getLoc(match_pinloc, getGridDimX(), getGridDimY());
-                BelId bel = getBelByLocation(loc);
-                if (bel == BelId()) {
-                    log_error("Pin %s not found\n", pinline.c_str());
-                }
-                std::string belname = getCtx()->nameOfBel(bel);
-                it->second->setAttr(IdString(ID_BEL), belname);
             } else {
-                log_error("Pin %s not found\n", pinname.c_str(this));
+                if (std::regex_match(pinline, match_pinloc, iobelre)) {
+                    // may be it's IOx#[AB] style?
+                    Loc loc = getLoc(match_pinloc, getGridDimX(), getGridDimY());
+                    BelId bel = getBelByLocation(loc);
+                    if (bel == BelId()) {
+                        log_error("Pin %s not found (TRBL style). \n", pinline.c_str());
+                    }
+                    std::string belname = getCtx()->nameOfBel(bel);
+                    it->second->setAttr(IdString(ID_BEL), belname);
+                } else {
+                    std::cout << "plain error:[" << pinline << "]" << std::endl;
```

Comment:
this looks like debugging that can be removed?

Reply:
Oops, how did I miss that?